src/AspNet.Security.OAuth.Bitrix24/Bitrix24AuthenticationConstants.cs

```diff
@@ -0,0 +1,258 @@
+﻿/*
```

Comment:
There's _a lot_ of constants in this file. Do they really all need to be built in?

Reply:
Well, they just represent available fields of user info endpoint. Do you think that they must not be there?

Reply:
The library doesn't _have_ to have _every_ possible thing built in, just the common defaults. Particularly as they're just strings and there's nothing stopping any integrator passing in literal strings, or making their own string constants classes with the extra ones they use/need in it.